internal/pkg/token/token.go

```diff
@@ -0,0 +1,54 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package token
+
+import (
+	"github.com/google/uuid"
+)
+
+const (
+	BootstrapTTL uuid.Time = 3600
```

Comment:
should this be controlled via userdata as setting?

Reply:
🤔 ... @bradbeam what do you think?

Reply:
I'm ok either way. I think we wanted to not give the user too much power over this just yet.

Reply:
we talked about it a bit. we'll start with hardcoded for now and adjust as needed. 1h seems generous enough to do an initial bootstrap of the cluster and that's really the only intent of the bootstrap token.